pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -18,7 +18,7 @@
 ## optional libraries
 
 , alsaSupport ? stdenv.isLinux, alsaLib
-, pulseaudioSupport ? true, libpulseaudio
+, pulseaudioSupport ? stdenv.isLinux, libpulseaudio
```

Comment:
Commit message should be descriptive like "firefox: disable pulseaudio support on non-Linux by default"